src/StochSystem.jl

```diff
@@ -48,8 +48,8 @@ end
 Concatenates the deterministic and stochastic parameter vectors `pf` and `pg` of a StochSystem `sys`.
 """
 function p(sys::StochSystem)
-    [sys.pf, sys.pg]
-end;
+    isnothing(sys.pg) ? sys.pf : [sys.pf, sys.pg]
```

Comment:
I think this would break things the way we write the deterministic part `sys.f`,
```julia
function f(u,p,t)
    a, b, ... = p[1]
    ...
end

pf = [a, b, ...]
sys = StochSystem(f, pf, u0)
```
So `p(sys)` should always return a Vector of Vectors.

The issue with `StochSystem(ds::CoupledODEs)` has been solved with PR #20.